evalml/pipelines/components/transformers/preprocessing/__init__.py

```diff
@@ -6,7 +6,8 @@
 from .natural_language_featurizer import NaturalLanguageFeaturizer
```

Comment:
PEP-8 recommends absolute paths for imports.  Any reason why these are relative?

Reply:
I think this should be fixed now!  Thanks